assets/js/blocks/rating-filter/block.tsx

```diff
@@ -293,20 +331,146 @@ const RatingFilterBlock = ( {
 	return (
 		<>
 			<div
-				className={ classnames( 'wc-block-rating-filter', {
-					'is-loading': isLoading,
-				} ) }
+				className={ classnames(
+					'wc-block-rating-filter',
+					`style-${ blockAttributes.displayStyle }`,
+					{
+						'is-loading': isLoading,
+					}
+				) }
 			>
-				<CheckboxList
-					className={ 'wc-block-rating-filter-list' }
-					options={ displayedOptions }
-					checked={ checked }
-					onChange={ ( item ) => {
-						onClick( item.toString() );
-					} }
-					isLoading={ isLoading }
-					isDisabled={ isDisabled }
-				/>
+				{ blockAttributes.displayStyle === 'dropdown' ? (
+					<>
+						<FormTokenField
+							key={ remountKey }
+							className={ classnames( borderProps.className, {
+								'single-selection': ! multiple,
+								'is-loading': isLoading,
+							} ) }
+							style={ {
+								...borderProps.style,
+								borderStyle: 'none',
+							} }
+							suggestions={ displayedOptions
+								.filter(
+									( option ) =>
+										! checked.includes( option.value )
+								)
+								.map( ( option ) => option.value ) }
+							disabled={ isLoading }
+							placeholder={ __(
+								'Select Rating',
```

Comment:
I don't have a strong opinion. But if we follow [these rules](https://github.com/WordPress/gutenberg/blob/086b77ed409a70a6c6a6e74dee704851eff812f2/docs/contributors/documentation/copy-guide.md#five-pay-attention-to-capitalization), I think we should use sentence capitalization, here.
```suggestion
								'Select rating',
```

Reply:
I took the convention from Filter by Attribute, but I agree with you. I changed both [in this commit](https://github.com/woocommerce/woocommerce-blocks/pull/7771/commits/6a09198ccc997eb68fd47c5f4445f88ad5ca916d).

Reply:
Hmmm, thinking about it twice, maybe I would undo this last commit (sorry for the back and forth :sweat: ). In the case of attributes, we can't know if the attribute name needs to be capitalized (ie: if the attribute is `CD style` we wouldn't want to lowercase it to `Select cd style`). 

And then, if we leave the attribute name capitalized, I see that it makes sense to keep `Select Rating` consistent.